R/parallel-config.R

```diff
@@ -14,5 +14,18 @@ find_parallel <- function(path, load_package = "source", package = NULL) {
     return(FALSE)
   }
 
-  identical(toupper(desc$get_field("Config/testthat/parallel", "")), "TRUE")
+  par <- identical(
+    toupper(desc$get_field("Config/testthat/parallel", default = "FALSE")),
+    "TRUE"
+  )
+
+  if (par) {
+    ed <- as.integer(desc$get_field("Config/testthat/edition", default = 2L))
+    if (ed < 3) {
+      inform("Running tests in parallel requires the 3rd edition")
+      want_parallel <- FALSE
```

Comment:
Should be `return(FALSE)` ?

Reply:
oh, sorry, it should be 
```
par <- FALSE
```

good catch.

Reply:
should this be a warning, btw?

Reply:
Anyway, should be good now.

Reply:
I don't feel too strongly about it either way

Reply:
Me neither, so let's leave it as is. :)